zerver/migrations/0455_set_default_for_can_mention_group.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 4.2.1 on 2023-06-12 10:47
+
+from django.db import migrations
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+def set_default_value_for_can_mention_group(
+    apps: StateApps, schema_editor: BaseDatabaseSchemaEditor
+) -> None:
+    Realm = apps.get_model("zerver", "Realm")
+    UserGroup = apps.get_model("zerver", "UserGroup")
+
+    groups_to_update = []
+    for realm in Realm.objects.all():
+        # The default value of `can_mention_group` is everyone system_group.
```

Comment:
I guess to be concrete, this implements the ability to mention role system groups? I think there was an issue for that...